build-all.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-for v in "2022.1.4" "2022.2.4" "2022.3.1"; do
+for v in "2022.1.4" "2022.2.4" "2022.3.1" "2022.3.2"; do
```

Comment:
Only the last bug fix version of each supported minor versions should be tested, so could you please update 2022.3.1